### DISCUSSION TITLE
Functions in dashboard UI won't return any result

### DISCUSSION THREAD

**Discussion by cjmling** [supabase/supabase#29178] [Questions]
I'm new to supabase and i was playing with functions, but query I expect to return result will shows no result (It return result in supabase clientjs).  For example this basic greet is not returning any result.

![image](https://github.com/user-attachments/assets/e6eb691d-533f-4150-9c06-bb45c9c39d3c)

**Comment by GaryAustin1** [comment]
As silentworks mentioned on Discord you are not running the function there, just creating it.

Do `select greet();` after you have run the code you show.
